zoloto/utils.py

```diff
@@ -2,6 +2,14 @@
 from functools import wraps
 from typing import Any, Callable, TypeVar
 
+try:
+    import ujson
+
+    encode_as_json = ujson.dumps
+except ImportError:
+    encode_as_json = json.dumps  # type: ignore
```

Comment:
I'm curious about why a `type: ignore` is necessary here?

Reply:
I believe the type signatures for `ujson.dumps` and `json.dumps` are different.

Reply:
Hmm, although given that, maybe we should facade both implementations in a common middle-ground :thinking:

Reply:
uJSON Stub: https://github.com/python/typeshed/blob/1e0e40f61c8a4c7f905c26fcf471763c2b33257f/third_party/2and3/ujson.pyi#L16

JSON Stub: https://github.com/python/typeshed/blob/1e0e40f61c8a4c7f905c26fcf471763c2b33257f/stdlib/3/json/__init__.pyi#L8

Yeah those are different. Should be simple enough to make a quick facade to avoid the `type: ignore`. Pretty sure you could annotate both with `Callable[[Any], str]`